src/app/app.component.ts

```diff
@@ -1,10 +1,135 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { bookType } from './book-type.enum';
+import { Author } from './models/author-model';
+import { Book } from './models/book-model';
+import { Bookstore } from './models/bookstore-model';
+import { Transaction } from './models/transaction-model';
+import { User } from './models/user-model';
+import { Wallet } from './models/wallet-model';
+import { BookstoreService } from './services/bookstore.service';
+import { UserService } from './services/user.service';
+import { transactionType } from './transaction-type.enum';
+import { Price } from './models/price-model';
+import { CurrencyType } from './currency-type.enum';
+import { Guid } from 'guid-typescript';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'bookstore-app';
+  public bookstore = new Bookstore();
+  public book1 = new Book(
+    Guid.create(),
+    'Angular 8',
+    bookType.Comedy,
+    new Author(Guid.create(), 'Janusz', 'Malarz'),
+    50,
+    10
+  );
+  public book2 = new Book(
+    Guid.create(),
+    'Alicja w Krainie Czarów',
+    bookType.Criminal,
+    new Author(Guid.create(), 'Janusz', 'Niewiem'),
+    20,
+    10
+  );
+  public book3 = new Book(
+    Guid.create(),
+    'Dzieci z Bul',
+    bookType.Romantic,
+    new Author(Guid.create(), 'Kuba', 'Koś'),
+    20,
+    5
+  );
+  public book4 = new Book(
+    Guid.create(),
+    'JavaScript',
+    bookType.Comedy,
+    new Author(Guid.create(), 'Bogdan', 'Malarz'),
+    20,
+    5
+  );
+  public books = [this.book2, this.book3, this.book4];
+
+  public user1 = new User(
+    Guid.create(),
+    'Jan Maj',
+    'maj@gmail.com',
+    new Wallet(new Price(123, CurrencyType.dollar))
+  );
+  public user2 = new User(
+    Guid.create(),
+    'Ania Lewangolska',
+    'ania@gmail.com',
+    new Wallet(new Price(650, CurrencyType.dollar))
```

Comment:
Wallet don't have price (at least user wallet) ;) I though you will add this Price object to Book.price. If you really want to leave this object for wallet too, rename 'Price' to something that will match both book.price and wallet.x.